### PR TITLE
appboard(ch04s07): Deploy Quarkus Microservices

### PR DESCRIPTION
**Pull request by shatakshi06**

### CODE REVIEW

**Comment by jramcast** [comment]
Adding @m-czernek . He's also participating in this chapter

File: deploy-review/app.adoc [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
+* Deploy the `Speaker microservice` using the `Quarkus openshift extension`.
+* Deploy the `Schedule Microservice` using the `Container Image deployment wizard` from the OpenShift Container Platform console.
+
+== Proposed App Flow
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
+* Deploy the `Session microservice` on the cluster using the `Quarkus openshift extension`.
+* Using the `Speaker microservice` with `Service Discovery on Openshift` with `Smallrye Stork`.
```
Comment (jramcast) [inline deploy-review/app.adoc:18]: @psolarvi might have something to say here

File: deploy-review/app.adoc [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+== Present App FLow
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
+* Deploy the `Speaker microservice` using the `Quarkus openshift extension`.
+* Deploy the `Schedule Microservice` using the `Container Image deployment wizard` from the OpenShift Container Platform console.
+
+== Proposed App Flow
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
+* Deploy the `Session microservice` on the cluster using the `Quarkus openshift extension`.
```
Comment (jramcast) [inline deploy-review/app.adoc:17]: Probably you want to align with @m-czernek here

File: deploy-review/app.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+== Present App FLow
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
+* Deploy the `Speaker microservice` using the `Quarkus openshift extension`.
+* Deploy the `Schedule Microservice` using the `Container Image deployment wizard` from the OpenShift Container Platform console.
+
+== Proposed App Flow
+
+* Logging in to the `Openshift Container Platform` and start using the project created for you.
```
Comment (jramcast) [inline deploy-review/app.adoc:16]: What does `start using the project` mean? Can we provide a bit more detail?

Comment (shatakshi06) [reply deploy-review/app.adoc:16]: Before deploying the application on Openshift, we login to Openshift and create a project to hold our application with the prefix deploy-review.

File: deploy-review/app.adoc (hunk 1 above)
Comment (psolarvi) [reply deploy-review/app.adoc:18]: I'm using the expenses microservice for SB, but TBO, any app that uses an external service supported by the Serbice Binding Operator (eg a database) will be OK. We only need to be sure that we pick the correct service. My recommendation is to use postgres as the persistence layer.

Comment (psolarvi) [reply deploy-review/app.adoc:18]: BTW, please update the line.. we are not using Stork and service discovery.. we are using Service Binding

**Review by m-czernek — APPROVED** [review]
From the deployment perspective, it's a +1. Please merge only after others' are satisfied with their requirements.

**Pull request merged**